src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -1427,6 +1444,12 @@ void Transformations::MainSnippets() {
                     return true;
                 if (output_shape[1].is_dynamic() || output_shape[1].get_length() > 256)
                     return true;
+                // Ticket 168474: BF16/FP16 MatMul with CopyB is not efficient enogh to be tokenized
```

Comment:
Minor:
`enogh`->`enough`
`MatMul` -> `FC`

Reply:
Corrected, thank you